WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/StatsListViewModel.kt

```diff
@@ -144,7 +162,33 @@ class InsightsListViewModel
         analyticsTracker,
         dateSelectorFactory.build(INSIGHTS),
         popupMenuHandler
-)
+) {
+    override val uiModel: LiveData<UiModel> = super.uiModel.map { model ->
```

Comment:
I think this belongs to the `org.wordpress.android.ui.stats.refresh.lists.UiModelMapper#mapInsights` method cause that's where we build the `UiModel`. You can either pass the `mutableNavigationTarget` as a parameter to the `mapInsights` function. What do you think?

Reply:
Good point!